mvrx/src/main/kotlin/com/airbnb/mvrx/MvRxLifecycleAwareFlow.kt

```diff
@@ -0,0 +1,70 @@
+@file:Suppress("EXPERIMENTAL_API_USAGE")
+
+package com.airbnb.mvrx
+
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.LifecycleEventObserver
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.lifecycleScope
+import kotlinx.coroutines.SupervisorJob
+import kotlinx.coroutines.channels.Channel
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.channelFlow
+import kotlinx.coroutines.flow.launchIn
+import kotlinx.coroutines.flow.onEach
+import kotlinx.coroutines.launch
+
+/**
+ * Similar to LifecycleScope.launchWhenStarted but cancels the block when the owner is stopped.
+ * This function suspends until the next time the owner is stopped.
```

Comment:
`suspends until the next time the owner is stopped` is this right? should this be "started"?